views/whereabouts/scheduledMoves.njk

```diff
@@ -97,15 +89,3 @@
     </div>
 
 {% endblock %}
-
-{% block bodyEnd %}
```

Comment:
Removing for now as it's breaking the table styles. I'm going to fix it in another PR.